app/src/components/avatar/Avatar.css

```diff
@@ -36,3 +36,9 @@
   box-shadow: none;
   border: 1px solid var(--color-grey-2);
 }
+.Avatar.xxsmall {
+  width: 30px;
+  height: 30px;
+  box-shadow: none;
+  border: 1px solid var(--color-grey-2);
+}
```

Comment:
Hmm... I think something is going on with the linting/husky again -_-

Reply:
:pray: thanks, this keeps happening, I've worked out now it only happens when committing through the VSCode source control GUI...
It seems for some reason the source control GUI is using a different shell to the bash I'm using in VSCode's CLI as the PATHS are different.
In my local shell I can add /home/<me>/.nvm/versions/node/v12.20.1/bin to the PATH variable but I'm not sure where the GUI is getting it's PATH from... 
For now I'll have to remember to stick to the CLI